allennlp/pretrained.py

```diff
@@ -37,6 +37,13 @@ def bidirectional_attention_flow_seo_2017() -> predictors.BidafPredictor:
                                 'machine-comprehension')
         return model.predictor() # type: ignore
 
+def naqanet_dua_2019() -> predictors.BidafPredictor:
+    with warnings.catch_warnings():
+        warnings.simplefilter(action="ignore", category=DeprecationWarning)
+        model = PretrainedModel('https://s3-us-west-2.amazonaws.com/allennlp/models/naqanet-2019.03.01.tar.gz',
+                                'machine-comprehension')
```

Comment:
This is almost certainly a deficit in my python knowledge but should this return statement be inside the `catch_warnings()` scope? Does that scope survive outside of the function?

Reply:
I don't think a scope can ever survive outside the function it's defined in

Reply:
And the warning that's being caught is getting thrown by the constructor, so we don't need to worry about it, anyway.